doc/main.html

```diff
@@ -436,6 +438,28 @@ <h4>Foreword section</h4>
 <pre>&lt;section id=&quot;sec-foreword&quot;&gt;
   &lt;p&gt;This is the additional information relevant to the document.&lt;/p&gt;
 &lt;/section&gt;
+</pre>
+        </div>
+
+        <p>When <code>pubType</code> is set to <code>RDD</code>, the <a>Foreword</a> section shall be present and there there shall be a single <code>dl</code>, after any author-supplied prose if supplied, which contains contact information about the proponent(s) of the RDD document. 
```

Comment:
I would create a section for RDDs

Reply:
Updated as suggested in https://github.com/SMPTE/html-pub/pull/315/commits/4276f14ad73bb02f932bac5d464ec691179dd1ab